packages/table/src/cell/cell.tsx

```diff
@@ -64,6 +64,26 @@ export interface ICellProps extends IIntentProps, IProps {
      * @default false
      */
     wrapText?: boolean;
+
+    /**
+     * Allows for setting a tab index on the cell, so the cell can be browser-focusable.
+     */
+    tabIndex?: number;
+
+    /**
+     * Callback to be called when cell is focused and key is pressed down.
```

Comment:
Articles or no articles? That is the question. Whether 'tis nobler in (the?) mind to suffer...

Reply:
You should probably add to this comment whether or not these callbacks are called while the cell is being edited.

Reply:
They are... but since you're in a text field they're ignored. These are used by the hotkeys -- i.e. this is the hack, passing the props down to the actual dom element.